Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/common/certificate/DccData.kt

```diff
@@ -1,8 +1,35 @@
 package de.rki.coronawarnapp.covidcertificate.common.certificate
 
+import de.rki.coronawarnapp.covidcertificate.common.decoder.RawCOSEObject
+
 data class DccData<CertT : DccV1.MetaData>(
     val header: DccHeader,
     val certificate: CertT,
     val certificateJson: String,
-    val kid: String
-)
+    val kid: String,
+    val rawCOSEObject: RawCOSEObject
+) {
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
```

Comment:
How come we need a custom implementation?

Reply:
because we pass ByteArray now ,any suggestions ?

Reply:
ok, haven't tested comparisons of ByteArray. You may be right

Reply:
Alternatively a `ByteString` could be used (if it makes sense), not sure atm what the other classes do with the signature.